merge_request_approvals.go

```diff
@@ -60,7 +61,7 @@ type MergeRequestApproverGroup struct {
 // GitLab API docs:
 // https://docs.gitlab.com/ee/api/merge_request_approvals.html#project-level-mr-approvals
 type MergeRequestApproverUser struct {
-	User struct {
+	ApproverUser struct {
```

Comment:
This will break code that uses `MergeRequestApproverUser` as now the expected (`User`) field name changed (to `ApprovedUser`). So please revert this change.